src/org/musicbrainz/webservice/impl/HttpClientWebServiceWs2.java

```diff
@@ -1,12 +1,6 @@
 package org.musicbrainz.webservice.impl;
 
-import org.apache.http.Header;
-import org.apache.http.HttpEntity;
-import org.apache.http.HttpEntityEnclosingRequest;
-import org.apache.http.HttpRequest;
-import org.apache.http.HttpResponse;
-import org.apache.http.HttpStatus;
-import org.apache.http.NoHttpResponseException;
+import org.apache.http.*;

```

Comment:
Let's keep the changes as minimal as possible. So, please reconfigure your IDE to not touch the imports.

Reply:
Done, thanks for the link!